syncer/syncer.go

```diff
@@ -3177,7 +3188,7 @@ func (s *Syncer) adjustGlobalPointGTID(tctx *tcontext.Context) (bool, error) {
 			zap.String("adjusted_gtid", gs.String()), zap.Error(err))
 		return false, err
 	}
-	s.checkpoint.SaveGlobalPoint(location)
+	s.saveGlobalPoint(location)
```

Comment:
what's the purpose of this change

Reply:
to ensure that there is only one function in the SaveGlobalPoint and this doesn't seem to break the original logic?

please correct me if I'm wrong😉